src/patterns/components/accordions/collection.yaml

```
@@ -5,4 +5,8 @@ information:
   - Content length inside the Accordion does not have a limit.
   - Title content of the Accordion can wrap, but it is recommended that it is short and to the point.
   - Accordions take up 100% width of their parent container until they hit their max width of 850px.
+restrictions:
+  - The data-id property is provided as a hook for automated tools. If you have multiple 
+    instances of the same variant of a component on the same page, make sure each instance 
+    has a unique data-id property. ("accordion-primary-1", "accordion-primary-2", "accordion-secondary-1", etc)
```

I like this definition. Thought, I think we want the parens inside the full stop. 
```suggestion
    has a unique data-id property ("accordion-primary-1", "accordion-primary-2", "accordion-secondary-1", etc).
```